docs/tutorials/ssp_profile_catalog_authoring/ssp_profile_catalog_authoring.md

```diff
@@ -227,15 +270,33 @@ Add prose here for required Section: Needed Extra
 
 ## Control A Guidance
 
-This is A Guidance.
+Control A prose
+
+### A Subpart
+
+A subpart prose
+
+#### A Subsubpart
+
+A subsubpart prose
+
+### B Subpart
 
-## Control B Guidance
+B subpart prose
 
-This is B Guidance.
+## Part a.
+
+### a by_id subpart
+
+a by_id subpart prose
 
```

Comment:
Part a. needs to be explained. The prose here belongs to subpart a. of the control statement.

Reply:
Changed to ```## Part a.

### Subpart of top level Part a. of the control

Prose added as a sub-part to Part a. of the control
```